mex/extractors/consent_mailer/settings.py

```diff
@@ -0,0 +1,39 @@
+from pydantic import Field, SecretStr
+
+from mex.common.settings import BaseSettings
+from mex.common.types import AssetsPath
+
+
+class ConsentMailerSettings(BaseSettings):
+    """Settings definition class for the consent mailer."""
+
+    mailpit_api_url: str = Field(
+        "localhost:8025",
+        description="The url to the api endpoint for mailpit. USED FOR TESTS ONLY!",
+    )
+    mailpit_api_user: SecretStr = Field(
+        SecretStr("user"),
+        description="The username used for Basic HTTP auth against the mailpit api."
+        " USED FOR TESTS ONLY!",
+    )
+    mailpit_api_password: SecretStr = Field(
+        SecretStr("password"),
+        description="The password used for Basic HTTP auth against the mailpit api."
+        "USED FOR TESTS ONLY!",
+    )
+    schedule: str = Field(
+        "0 0 * * *",
```

Comment:
let's use a safer default, so we don't accidentally launch a daily spam bot

```suggestion
        "0 0 30 2",  # Defaults to Feb 30, aka "never"
```

Reply:
"0 0 30 2" is not valid. expecting u mean "0 0 30 2 *"

Reply:
Seems dagster is validating this aswell. I change it to a far future 🤖

Reply:
since dagster is validating the schedule and 30.2 is invalid i have to change it to "0 0 29 2 *". Any other ideas?

Reply:
dagster uses croniter under the hood and that does allow for a seventh slot to set a year
https://github.com/pallets-eco/croniter?tab=readme-ov-file#about-year

-> `0 0 1 1 * 0 1970` should work as a sane default that will never happen (at least not for the next 9945 years)

Reply:
Seems like dagster doing some extra validation.
`
Found invalid cron schedule '0 0 1 1 * 0 1970' for schedule 'None''. Dagster recognizes standard cron expressions consisting of 5 fields.
`

Reply:
it is also possible to not set a schedule at all. This is how I did it for Datenkompass. If that is an option?

Reply:
How did u achieve this? When I set it to an empty string, dagster also complains :(
`Found invalid cron schedule '' for schedule 'None''. Dagster recognizes standard cron expressions consisting of 5 fields.`